apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js

```diff
@@ -23,7 +24,10 @@ function HelpCenterContent() {
 	const [ showHelpIcon, setShowHelpIcon ] = useState( false );
 	const { setShowHelpCenter } = useDispatch( 'automattic/help-center' );
 	const [ showHelpIconDot, setShowHelpIconDot ] = useState( false );
-	const { data, isLoading } = useHasSeenWhatsNewModalQuery( window._currentSiteId );
+	const isSimpleSite = useSelector( ( state ) => getIsSimpleSite( state ) ) || false;
```

Comment:
You can write this as `const isSimpleSite = useSelector( getIsSimpleSite ) || false;`